### ISSUE TITLE
ComfyUI Error Report - WanVideoSampler failed at the final step

### ISSUE THREAD

**Issue by ivever2** [kijai/ComfyUI-WanVideoWrapper#214]
## Error Details
- **Node ID:** 27
- **Node Type:** WanVideoSampler
- **Exception Type:** AssertionError
- **Exception Message:** 
## Stack Trace
```
  File "F:\Ai\ComfyUI_windows_portable\ComfyUI\execution.py", line 327, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "F:\Ai\ComfyUI_windows_portable\ComfyUI\execution.py", line 202, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "F:\Ai\ComfyUI_windows_portable\ComfyUI\execution.py", line 174, in _map_node_over_list
    process_inputs(input_dict, i)

  File "F:\Ai\ComfyUI_windows_portable\ComfyUI\execution.py", line 163, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "F:\Ai\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-WanVideoWrapper\nodes.py", line 1785, in process
    temp_x0 = sample_scheduler.step(
              ^^^^^^^^^^^^^^^^^^^^^^

  File "F:\Ai\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-WanVideoWrapper\wanvideo\utils\fm_solvers_unipc.py", line 721, in step
    assert self.this_order > 0
           ^^^^^^^^^^^^^^^^^^^

```
## System Information
- **ComfyUI Version:** 0.3.26
- **Arguments:** ComfyUI\main.py --output-directory F:\Ai\Output-SYNC\txt2img-images\
- **OS:** nt
- **Python Version:** 3.11.8 (tags/v3.11.8:db85d51, Feb  6 2024, 22:03:32) [MSC v.1937 64 bit (AMD64)]
- **Embedded Python:** true
- **PyTorch Version:** 2.6.0+cu124
## Devices

- **Name:** cuda:0 NVIDIA GeForce RTX 4090 : cudaMallocAsync
  - **Type:** cuda
  - **VRAM Total:** 25756696576
  - **VRAM Free:** 15286703744
  - **Torch VRAM Total:** 8589934592
  - **Torch VRAM Free:** 29922944

**Comment by kijai** [comment]
I think that's a bug with the unipc scheduler if you do lots of steps, added possible fix.
